torch/_torch_docs.py

```diff
@@ -1500,6 +1500,58 @@ def merge_dicts(*dicts):
     False
 """)
 
+add_docstr(torch.cov,
+           r"""
+cov(input1, input2=None, rowvar=True, bias=None, ddof=None, fweights=None, aweights=None) -> Tensor
```

Comment:
cov(input, other, rowvar, bias, ddof, fweights, aweights)

Reply:
I wonder if we should require that arguments after other be kwarg-only

Reply:
The args might be frequently used, it would provide better clarity this way I believe..